### PR TITLE
Added script to download PR descriptions + 2.1 release notes.

### PR DESCRIPTION
**Pull request by joakim-hove**
This script will spin through the PR descriptions and create a draft file which can be used to create release notes. Observe that before you can use this script you need to create a GitHub access token: https://help.github.com/articles/creating-an-access-token-for-command-line-use/

### CODE REVIEW

**Comment by pgdr** [comment]
Please follow the `__name__ == '__main__'` idiom in case we ever want to use the script from somewhere else.

File: script/download-pr [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+closed_pr = set( [1197 , 1201, 1235, 1244, 1245, 1252, 1261, 1262, 1366,
+                  1357, 1282, 1279, 1267, 1273, 1305, 1312, 1313, 1339 , 1346,
+                  1354, 1370,1371, 1381])
+
+def getPRList( api_token, first_pr ):
+    pr = first_pr
+    pr_list = []
+    prev_len = 0
+    sys.stderr.write("Downloading PR: ")
```
Comment (pgdr) [inline script/download-pr]: why `stderr`?

Comment (joakim-hove) [inline script/download-pr]: Hmmm - good question?

**Comment by joakim-hove** [comment]
> merge at will

Thank you - I am currently manually creating release-notes based on the rawdata from this script. The intention was to merge that along with the script.

**Pull request merged**